<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphView</class>
 <widget class="QDialog" name="GraphView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>621</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>821</width>
    <height>621</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>821</width>
    <height>621</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>621</width>
    <height>621</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>621</width>
    <height>621</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>График</string>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>619</width>
     <height>619</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="make_button">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>470</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Make Graph</string>
   </property>
  </widget>
  <widget class="QPushButton" name="x_max_add10">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>280</y>
     <width>45</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>+10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="x_max_line">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>240</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X max</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="x_max_add25">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>330</y>
     <width>45</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>+25</string>
   </property>
  </widget>
  <widget class="QPushButton" name="x_max_add100">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>380</y>
     <width>45</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>+100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="x_max_sub100">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>380</y>
     <width>45</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>-100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="x_max_sub10">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>280</y>
     <width>45</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>-10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="x_max_sub25">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>330</y>
     <width>45</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>-25</string>
   </property>
  </widget>
  <widget class="QPushButton" name="x_min_sub25">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>45</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>-25</string>
   </property>
  </widget>
  <widget class="QPushButton" name="x_min_add10">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>60</y>
     <width>45</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>+10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="x_min_add100">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>160</y>
     <width>45</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>+100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="x_min_sub100">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>160</y>
     <width>45</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>-100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="x_min_sub10">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>45</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>-10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="x_min_line">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X min</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="x_min_add25">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>110</y>
     <width>45</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>+25</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return_button">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>530</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Return to calc</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
